hazelcast/src/main/java/com/hazelcast/instance/DefaultAddressPicker.java

```diff
@@ -412,23 +411,21 @@ public int hashCode() {
     }
 
     private class AddressDefinition extends InterfaceDefinition {
+
         InetAddress inetAddress;
         int port;
 
-        private AddressDefinition() {
```

Comment:
Same for AddressDefinition. I also removed the private keyword from the other constructors for the same reason.